packages/esm-sms-app/src/providers/modals/upload-template-modal.scss

```diff
@@ -0,0 +1,33 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@import '@openmrs/esm-styleguide/src/vars';
+
+.errorContainer {
+  margin: spacing.$spacing-05 0;
+}
+
+.uploadFile {
+  color: $ui-02;
+  width: 100%;
+  height: 406px;
```

Comment:
Why is this the value? Why is it specified in `px` rather than `rem` (which is our standard for anything larger than 1px, generally)

Reply:
fixed